src/i18n/en/translation.en.json

```diff
@@ -1089,5 +1089,13 @@
   "associates-view": {
     "less": "show less",
     "more": "...plus {{count}} more"
+  },
```

Comment:
I think that for the text to have meaningful content then it has to be specific to each tab that is shown. So I think there will need to be a translation section for each entity.tab. 

So there would need to be a hubs, users, organizations, authorization section underneath the admin level etc., with tabs below each of those. Make sense?

Reply:
I updated the structure.
So now, whenever one comes up with a custom explanation for a tab under an entity, he just copies the part of the structure from "generic" under e.g. "hub" and edits it there:
```
    "hub": {
      "profile": { "explanation": "This is a very specific message about the Hub Profile." }
      ...
    }

```
Does it work for you, @techsmyth ?

Reply:
Yes, thanks - this works...